### PR TITLE
doc/development: add note to run individual blackbox tests

### PR DESCRIPTION
**Pull request by rfairley**
Minor note, which I found useful.

### CODE REVIEW

**Comment by bgilbert** [comment]
ok to test

File: doc/development.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ HELPERS=HOST ./build_blackbox_tests
 sudo sh -c 'PATH=$PWD/bin/amd64:$PATH ./tests.test'
 ```
 
+To run a subset of the blackbox tests, pass a regular expression into `-test.run`. As an example:
+
+```
+sudo sh -c 'PATH=$PWD/bin/amd64:$PATH ./tests.test -test.run TestIgnitionBlackBox/Preemption.*'
```
Comment (ajeddeloh) [inline doc/development.md]: Might add that you can get a list of tests with `-list`, otherwise LGTM.

Comment (rfairley) [reply doc/development.md]: :+1: - adding this in

**Comment by rfairley** [comment]
^ updated!

**Pull request merged**